Corona-Warn-App/src/main/res/layout/include_information_details.xml

```diff
@@ -64,7 +64,7 @@
             android:layout_marginStart="@dimen/spacing_normal"
             android:layout_marginTop="@dimen/spacing_medium"
             android:layout_marginEnd="@dimen/spacing_normal"
-            android:autoLink="all"
+            android:autoLink="web|email"
```

Comment:
Could you please check if this affecting `phone` number?

Reply:
As written in the description I had a look for phone numbers in the text and could not see any. If there are any phone numbers in the text I would add it, otherwise I think we should stick only with the necessary categories. Views containing phone numbers (like the 116 117 hotline or the technical hotline) are separated and use `autoLink="phone"`

Reply:
this one: the layout  is included  in many other places, have you checked them all? and did you check the locales too?
For example the legal text different variants for DE, EN, TR.

<img width="518" alt="Screenshot 2021-02-04 at 12 16 37" src="https://user-images.githubusercontent.com/25054729/106885581-17552f00-66e3-11eb-8deb-38e718847062.png">

Reply:
Good point, I did not check all of these. The translations should not play a role as I would expect the content to be the same (I mean why should TR translation contain a phone number which en translation does not at all?). But I will check them later and ping you again.

Reply:
@mtwalli I just double checked it. `phone` is what is actually causing the issues especially on the Terms of Use screen.
Then I checked every screen (in German) for phone numbers:

fragment_information_about: None
fragment_information_contact: Technical hotline is a completely separated view (with a click listener in the code) and the `116 117` is in a separate TextView which uses `android:autoLink="phone"` so it still works. No phone number in the text of `include_information_details` here.
fragment_information_legal: None
fragment_information_privacy: None
fragment_information_technical: None
fragment_information_terms: None
fragment_main_overview: None
fragment_main_share: None
fragment_settings_background_priority: None
fragment_settings_notifications: None
fragment_settings_reset: None
fragment_settings_tracing: None

So for me it looks fine, assuming the translations do not introduce phone numbers on one of these screens which german texts do not have.

Reply:
In your branch the phone number  `116 117` is not working 
![device-2021-02-05-105603](https://user-images.githubusercontent.com/25054729/107018633-f8b56d80-67a0-11eb-94b7-79073d6d8b9f.png)

Reply:
Regarding this issue, I think it might be related to https://github.com/corona-warn-app/cwa-app-android/pull/2292/files#r570902849

Reply:
That's interesting, this was working on my phone. But as you mentioned it is not related to the changes made in this PR here. So we should keep it here as it is and fix it in the other mentioned PR. What do you think about it?

Reply:
I was able to reproduce it on an API 26 emulator and fixed it

Reply:
Hi guys @PhilippNowak96 / @mtwalli 
I just finished commenting in https://github.com/corona-warn-app/cwa-app-android/pull/2292 and I concluded that autoLink for phone numbers is quite sensitive to locale and Android version unfortunately.